devices/src/virtio/block.rs

```diff
@@ -75,6 +78,7 @@ enum ExecuteError {
 impl ExecuteError {
     fn status(&self) -> u32 {
         match self {
+            &ExecuteError::BadRequest(_) => VIRTIO_BLK_S_IOERR,
```

Comment:
nit: these lines can be rewritten as:
```rust
&ExecuteError::BadRequest(_) | &ExecuteError::Flush(_) | &ExecuteError::Read(_) | &ExecuteError::Seek(_) => VIRTIO_BLK_S_IOERR

```

Reply:
Yes, but I think it looks neater, each on its own line.